src/main/resources/locales/en/rpt-eastern-weekly-hearing-list.json

```diff
@@ -1,4 +1,4 @@
 {
-    "title": "Residential Property Tribunal: Eastern region Weekly Hearing List",
-    "heading": "Residential Property Tribunal: Eastern region Weekly Hearing List"
+    "title": "First-tier Tribunal (Residential Property Tribunal): Eastern region Weekly Hearing List",
+    "heading": "First-tier Tribunal (Residential Property Tribunal): Eastern region Weekly Hearing List"
```

Comment:
The word "region" for all RPT lists starts with lower case here but it is in upper case in pip-data-models. Don't mind which one should be used can can you keep them consistent?